.circleci/config.yml

```diff
@@ -1,73 +1,46 @@
-version: 2
-
-rspec: &rspec
-  steps:
-    - checkout
-    - run: bundle install
-    - run: rake spec
+version: 2.1
 
 jobs:
-  # Ruby 2.3
-  ruby-2.3-rspec:
-    docker:
-      - image: circleci/ruby:2.3
-    <<: *rspec
-
-  # Ruby 2.4
-  ruby-2.4-rspec:
-    docker:
-      - image: circleci/ruby:2.4
-    <<: *rspec
-
-  # Ruby 2.5
-  ruby-2.5-rspec:
-    docker:
-      - image: circleci/ruby:2.5
-    <<: *rspec
-
-  # JRuby
-  jruby:
-    docker:
-      - image: circleci/jruby:9
-    steps:
-      - checkout
-      - run: sudo apt-get install -y make
-      - run: bundle lock
-      - restore_cache:
-          keys:
-            - bundle-v2-{{ checksum "Gemfile.lock" }}
-            - bundle-v2-
-      - run: bundle install --path vendor/bundle
-      - save_cache:
-          key: bundle-v2-{{ checksum "Gemfile.lock" }}
-          paths:
-            - vendor/bundle
-      - run: rake spec
-
-  code-climate:
+  rake_default:
+    parameters:
+      image:
+        description: "Name of the Docker image."
+        type: string
+        default: "circleci/ruby"
     docker:
-      - image: circleci/ruby:2.5
+      - image: << parameters.image >>
+    environment:
+      # CircleCI container has two cores, but Ruby can see 32 cores. So we
+      # configure test-queue.
+      # See https://github.com/tmm1/test-queue#environment-variables
+      TEST_QUEUE_WORKERS: 2
+      JRUBY_OPTS: "--dev -J-Xmx1000M" # get more accurate coverage data in JRuby
```

Comment:
“get more accurate coverage data in JRuby” doesn’t apply.

Reply:
Oops. I missed it 💦